packages/core/src/lib/notify.svelte

```diff
@@ -0,0 +1,132 @@
+<svelte:options immutable />
+
+<script lang="ts">
+type Variants = 'danger' | 'warning' | 'success' | 'info';
+
+import cx from 'classnames';
+import { createEventDispatcher } from 'svelte';
+import { Button, Icon } from '$lib';
+
+export let title = '';
+export let message = '';
+export let variant: Variants = 'info';
+export let progress = 1;
+export let exitable = 'false';
+
+export const htmlToBoolean = (value: string | undefined, key: string) => {
```

Comment:
We can now accept boolean props so can you remove the htmlToBoolean?

Reply:
yupp